src/test/java/cat/udl/eps/softarch/academicrecruit/steps/JobApplicationStepsDefs.java

```diff
@@ -46,14 +48,20 @@ public void iRegisterANewJobApplicationNameRequirementsAndDescription(String nam
     }
 
     @And("It has been created a new job application with name {string}, requirements {string} and description {string}")
-    public void itHasBeenCreatedANewJobApplicationWithNameRequirementsAndDescription(String arg0, String arg1, String arg2) {
-    }
+    public void itHasBeenCreatedANewJobApplicationWithNameRequirementsAndDescription(String name, String requirements, String description) throws Exception {
+        List<JobApplication> applicantList = jobApplicationRepository.findByNameContaining(name);
 
-    @And("I can check the job application")
-    public void iCanCheckTheJobApplication() {
+        stepDefs.result = stepDefs.mockMvc.perform(
```

Comment:
This is a Then step checking for postconditions so no need to use the API, you can check existence using the findBy... method